site/projects/intro_timeseries.md

```diff
@@ -14,7 +14,7 @@ Potentially include Pandas (e.g. basic dataframes, block averages, moving window
 
 ### Analysis
 * Numpy (basics)
-* Pandas (basics)
+* [Pandas (basics)] (https://github.com/Unidata/pyaos-ams-2021/blob/master/notebooks/pandas.ipynb)
```

Comment:
The issues in the repo have not been updated yet (soon), so there is no way you would have known this one, but we are going to use nbviewer to visualize the notebooks rather than linking to the ipynb file directly in github.

To do that, basically change the first part of the link:

`https://github.com/Unidata/pyaos-ams-2021/`

to

`https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/`

the `blob/master/notebooks/analysis/pandas.ipynb` (once the notebook is moved) is the same between the two URLs :-) Once this is merged, then https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/analysis/pandas.ipynb should work (similar to [this](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/visualization/python-awips-WorkingWithSurfaceObs.ipynb) one).